server/app/src/main/java/com/ams/app/dto/SubjectStatsresponsedto.java

```diff
@@ -0,0 +1,18 @@
+package com.ams.app.dto;
+
+import java.sql.Timestamp;
+
+import lombok.Data;
+
+@Data
+public class SubjectStatsresponsedto {
+    private String date;
+    private String status;
+    private Timestamp time;
+    
+    public SubjectStatsresponsedto(String date, String status, Timestamp time) {
+        this.date = date;
+        this.status = status;
+        this.time = time;
+    }
```

Comment:
Why are we creating DTO manually? Remove this constructor and use @AllArgsConstructor if it's needed.

Reply:
Used @AllArgsConstructor